### DISCUSSION TITLE
How to allow ads?

### DISCUSSION THREAD

**Discussion by omidshojaee** [bootmortis/iran-hosted-domains#160] [Q&A] [answered]
Hello,

My client is v2rayN and I currently use ```ext:iran.dat:all``` as the rule.

For some reasons I prefer not to block ads.

So what should I use instead of ```ext:iran.dat:all``` ?

I think it is ```ext:iran.dat:ir``` AND ```ext:iran.dat:other```. Am I right?

**Comment by bootmortis** [comment, marked as answer]
There's no need to make any changes as `iran.dat:all` doesn't have any ads to begin with.

https://github.com/bootmortis/iran-hosted-domains#full-categories
